Azure-Samples/communication-services-dotnet-quickstarts
# ACS quickstart for JobRouter with AzureOpenAI

## Purpose
QuickStart guide to add Azure OpenAI into your Job Router ACS resource for matching.

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
Quick start application


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Clone repo setup resources as noted in Readme. Run application, check logs in Azure function to monitor output of OpenAI

## What to Check
Verify that the following are valid
* Check scores given to workers viewing the Azure Function logs.
